src/app/shared/resolvers/title.resolver.ts

```diff
@@ -0,0 +1,6 @@
+import { ResolveFn } from '@angular/router';
+
+export const title2Resolver: ResolveFn<string> = (route, state) => {
+  const slug = route.params['slug'];
+  return slug;
```

Comment:
```suggestion
  return route.params['slug'];
```
This can be returned directly.